cg/apps/gt.py

```diff
@@ -37,3 +47,33 @@ def upload(self, bcf_path: str, samples_sex: dict, force: bool=False):
                 analysis_obj.sex = samples_sex[analysis_obj.sample_id]['analysis']
                 analysis_obj.sample.sex = samples_sex[analysis_obj.sample_id]['pedigree']
                 self.commit()
+
+
+    def get_trending(self, sample_id : str = '' , days: str = '' ):
+        """Get trending object with one or many samples."""
+        trending_obj = None
+        trending_call = copy.deepcopy(self.base_call)
```

Comment:
There is no need to deepcopy a flat list

```suggestion
        trending_call = self.base_call[:]
```